en/subscription_services_privacy_notice.md

```diff
@@ -1,55 +1,68 @@
 # Mozilla Subscription Services | Privacy Notice
 
-Version 1.1, Effective January 10, 2024
-{: datetime="2024-01-10" }
+Version 1.2, Effective February 5, 2024
+{: datetime="2024-02-05" }
 
 ## At Mozilla, we design products with your privacy in mind.
 
 __Mozilla VPN__ protects your device’s internet connections. Mozilla partners with Mullvad to privately and securely encrypt your internet traffic.
 
-__Firefox Relay__ protects your identity with secure email and phone masking. Our secure and easy to use email and phone masks help keep your identity private so you can sign up for new accounts anonymously, stop spam texts and junk calls, and get only the emails you want in your inbox.  
+__Mozilla Monitor__ can monitor to see if any of your accounts have been in known data breaches. In some regions, it can also help you scan for and remove your information from places around the web. 
 
-__Firefox Monitor__ can monitor to see if any of your accounts have been in known data breaches. See [Firefox Monitor Terms and Privacy](https://www.mozilla.org/privacy/firefox-monitor/).
+__Firefox Relay__ protects your identity with secure email and phone masking. Our secure and easy to use email and phone masks help keep your identity private so you can sign up for new accounts anonymously, stop spam texts and junk calls, and get only the emails you want in your inbox.  
```

Comment:
Open question: I think this needs to add "In some regions" here, similarly to Monitor. Phone masking is only available in US.

Reply:
Confirmed with Luke Crouch. Can we please update to "__Firefox Relay__ protects your identity with secure email and, in some regions, phone masking."?